### PR TITLE
openbox: fix distfiles, checksum, add openbox-devel short_desc.

### PR DESCRIPTION
**Pull request by cr6git**
... :zipper_mouth_face:

### CODE REVIEW

**Comment by Vaelatern** [comment]
I will not accept this with third party untrusted un-upstreamed code, even as an optional build option.

**Comment by Vaelatern** [comment]
Unless another @voidlinux/pkg-committers wants to accept such things, I think we did with freetype (?) and it caused no end of trouble.

File: srcpkgs/openbox/template [hunk 1]
```diff
@@ -16,14 +16,24 @@ conf_files="
 short_desc="Standards compliant, fast, light-weight, extensible window manager"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2"
-homepage="http://www.openbox.org"
-distfiles="http://icculus.org/openbox/releases/$pkgname-$version.tar.gz"
-checksum=8b4ac0760018c77c0044fab06a4f0c510ba87eae934d9983b10878483bde7ef7
+homepage="http://openbox.org/wiki/Main_Page"
```
Comment (Vaelatern) [inline srcpkgs/openbox/template]: Let them redirect it. No reason to do this.

File: srcpkgs/openbox/template [hunk 2]
```diff
@@ -37,6 +47,7 @@ post_install() {
 }
 openbox-devel_package() {
 	depends="${makedepends} libopenbox>=${version}_${revision}"
+	short_desc="Openbox development files"
```
Comment (Vaelatern) [inline srcpkgs/openbox/template]: :+1:

File: srcpkgs/openbox/template (hunk 1 above)
Comment (cr6git) [reply srcpkgs/openbox/template]: OK.

Comment (cr6git) [reply srcpkgs/openbox/template]: OK!

**Comment by cr6git** [comment]
@Vaelatern I will never go against you. I removed rounded corners stuff.

**Pull request merged**